rust-toolchain.toml

```diff
@@ -1,5 +0,0 @@
-[toolchain]
-channel = "1.63"
```

Comment:
I thought we got rid of this file already. I noticed we didn't since there were some improvements in the `1.64` borrow checker I depend on but I see the code compiling on `nightly-2022-08-15`. So either bumping the version or removing the file works for me :)